SKILL.tsv

```diff
@@ -5734,302 +5734,302 @@ SKILL_20151102_005733	$[Merkabah] petrifies [Cursed] enemies for 3 seconds.
+SKILL_20151224_005986	Serpentine: Enhance
+SKILL_20151224_005987	Increases the damage dealt on an enemy with [Serpentine] by 1% per attribute level.
+SKILL_20151224_005988	Serpentine: Bleeding
+SKILL_20151224_005989	Afflicts [Bleeding] with 2% rate per attribute level of [Serpentine]for 8 seconds. Bleeding damage is proportional to the character's physical attack.
+SKILL_20151224_005990	Two-handed Spear Mastery: Large Target Hunter
+SKILL_20151224_005991	Increases the damage dealt on large enemies with [Two-handed Spear] by 4% per attribute level against normal enemies and 2% per attribute level against boss enemies.
+SKILL_20151224_005992	[Energy Bolt] damage is enhanced to Wizard circle 2 standard.
+SKILL_20151224_005993	[Energy Bolt] damage is enhanced to Wizard circle 3 standard.
+SKILL_20151224_005994	[Earthquake] damage is enhanced to Wizard circle 3 standard.
+SKILL_20151224_005995	Deals additional damage equal to 40% magic attack per attribute level when attacking an enemy under [Sleep] with [Energy Bolt].
+SKILL_20151224_005996	Enemies affected by [Lethargy] will receive 20% additional damage with Strike attacks.
+SKILL_20151224_005997	Increases magic damage by 10% per attribute level when [Quick Cast] is active.
+SKILL_20151224_005998	[Enchant Fire] will decrease the fire resistance of nearby enemies by 10 per attribute level for 5 seconds.
+SKILL_20151224_005999	Freeze duration of [Subzero Shield] is increased by 0.5 seconds per attribute level.
+SKILL_20151224_006000	Decreases the physical defense by 10% per attribute level on enemies that are pulled by [Gravity Pole].
+SKILL_20151224_006001	Decreases the AoE Defense Ratio of the enemies gathered through [Hangman's Knot] by 1.
+SKILL_20151224_006002	nemies affected by [Swell Body] will have their movement speed decreased by 15% and physical and magical attack increased by 20% per attribute level.
+SKILL_20151224_006003	Enemies affected by [Shrink Body] will have their movement speed increased by 10% and physical and magical attack decreased by 25% per attribute level.
+SKILL_20151224_006004	When you attack an enemy affected by [Swell Body] while [Left Arm Enhancement] is active, increases the damage dealt on an enemy by 35 per attribute level.
+SKILL_20151224_006005	When you attack an enemy affected by [Swell Body] while [Left Arm Enhancement] is active, it will continuously apply the [Attacked Weakened] status effect. Increases the duration by 8 seconds per attribute level and [Attacked Weakened] can stack up to 10 times.
+SKILL_20151224_006006	Evocation: Enhance
+SKILL_20151224_006007	Increases the damage dealt on an enemy with [Evocation] by 1% per attribute level.
+SKILL_20151224_006008	Desmodus: Enhance
+SKILL_20151224_006009	Increases the damage dealt on an enemy with [Desmodus] by 1% per attribute level.
+SKILL_20151224_006010	Critical hit resistance of the enemy fallen to [Slow] is decreased by 15% of the user's intelligence value.
```

Comment:
Enemies affected by [Slow] have their critical resistance decreased by 15% equal to by the character (caster) INT.